test/projects/file-generate/.gitignore

```diff
@@ -0,0 +1,77 @@
+# Temporary Build Files
```

Comment:
I wouldn't mind a README.md in this "project" that explains what we're using it for. Including mentioning that the only piece that's actually important right now is the two test.go files containing `go:generate` directives.